### PR TITLE
Fix code typos in Readme

### PR DESCRIPTION
**Pull request by mhammerc**
Hello!

First of all, great work on Vulkan-Hpp and Vulkan. I am only beginning in the world of Vulkan and 3D rendering, which lead me to read the readme and try one code snippet in my IDE to test auto-completion.

The code snippet I copy/pasted have a few typos, I think. I only tried these two examples, which mean I haven't checked other examples.
I hope nothing is wrong in the code change I propose.

This PR is really low-priority!

Thank you for your great work.

### CODE REVIEW

**Comment by mhammerc** [comment]
I think `vk::createImage` doesn't  exist as we create image from `device.createImage` ?

**Comment by asuessenbach** [comment]
What a shame, that our Readme has so many bugs!
Would you mind extending your PR with replacing the wrong `vkCreateImage`  and `vk::createImage` with `vkCreateInstance` and `vk::createInstance`, accordingly?

**Comment by mhammerc** [comment]
haha :p 

It is done! I also edited a few more things, I let you check it out.

File: README.md [hunk 1]
```diff
@@ -98,22 +98,22 @@ Especially the first one is hard to detect.
...
 
 ```c++
-vk::ImageCreateInfo ci({}, vk::ImageType::e2D, vk::format::eR8G8B8A8Unorm,
+vk::ImageCreateInfo ci({}, vk::ImageType::e2D, vk::Format::eR8G8B8A8Unorm,
                        { width, height, 1 },
-                       1, 1, vk::SampleCount::e1,
-                       vk::ImageTiling::eOptimal, vk::ImageUsage:eColorAttachment,
-                       vk::SharingMode::eExclusive, 0, 0, vk::Imagelayout::eUndefined);
+                       1, 1, vk::SampleCountFlagBits::e1,
+                       vk::ImageTiling::eOptimal, vk::ImageUsageFlagBit::eColorAttachment,
```
Comment (asuessenbach) [inline README.md:104]: Please add a trailing `s` to `ImageUsageFlagBit`.

Comment (mhammerc) [reply README.md]: Oh! Good catch! I thought I properly tested that 😂

**Comment by asuessenbach** [comment]
Thanks a lot for your help!

**Pull request merged**